### DISCUSSION TITLE
AUTOMATIC1111 is getting better

### DISCUSSION THREAD

**Discussion by HenkDz** [hlky/stable-diffusion-webui#690] [General]
Although this fork is advancing really good as well, the original fork just gotten even better.

![image](https://user-images.githubusercontent.com/17301927/188523934-34123f7d-17a9-4d68-8de0-61951e3c397c.png)

**Comment by mchaker** [comment]
I think they should merge and continue as a joint effort, with all features combined/available in one UI.

**Reply by HenkDz** [reply]
this one got messy a bit with many bugs. I think it's better for all of us if they stay separate.

**Comment by telemole** [comment]
> Although this fork is advancing really good as well, the original fork just gotten even better.
> 
> ![image](https://user-images.githubusercontent.com/17301927/188523934-34123f7d-17a9-4d68-8de0-61951e3c397c.png)

How can we run this now? I have the most recent version and I don't see these controls :( any pointer to some instructions for this install? thanks!

**Reply by HenkDz** [reply]
https://github.com/AUTOMATIC1111/stable-diffusion-webui

**Reply by Theagainmen** [reply]
How to update my current installation?

**Reply by HenkDz** [reply]
You don't, that's a separate repo with its won instructions for installing.
